src/components/BlockCreateModal.vue

```diff
@@ -16,8 +16,8 @@
       <BlockSchemaCreateForm v-if="blockSchema" :id="formId" :block-schema="blockSchema" hide-footer @submit="submit" />
     </template>
 
-    <template #actions>
-      <SubmitButton action="Create" :form-id="formId" />
```

Comment:
🤕 

i... wonder how this worked for me "sometimes" but 🤷 .

Reply:
Yeah same, it worked fine for me before. But sure seems like it shouldn't have ever worked.